Taskfile.yml

```diff
@@ -41,8 +41,7 @@ tasks:
   setup-win:
     platforms: [ windows ]
     cmds:
-      - powershell -Command "Invoke-WebRequest 'https://drive.google.com/uc?export=download&id=1I-hwc0RHoQqW774gbS5qR-GHa1E7BlsS' -OutFile {{.DATA_DIR}}/sena-3.zip -Resume"
-      - powershell -Command "Invoke-WebRequest 'https://drive.usercontent.google.com/download?export=download&id=1Jk-eSDho8ATBMS-Kmfatwi-MWQth26ro&confirm=t' -OutFile {{.DATA_DIR}}/elawa.zip -Resume"
+      - powershell -File download.ps1
```

Comment:
I would prefer for the download.ps1 script to take command-line parameters specifying the URL to download, the filename to download it to, and the checksum value. Then this would be two steps in the Taskfile.yml, just like setup-linux is two steps — and more importantly, it means that if we add more projects in the future, we'd only need to change a single file (Taskfile.yml) rather than needing to change Taskfile.yml *and* download.ps1.